### ISSUE TITLE
ModuleNotFoundError while importing MarketplaceApp

### ISSUE THREAD

**Issue by drvgautam** [materials-marketplace/python-sdk#26]
`from marketplace.app import MarketPlaceApp`

Error message:
```from utils import camel_to_snake 
ModuleNotFoundError: No module named 'utils' ```

**correction hint:** check  `from utils import camel_to_snake` inside marketplace/app/__init__.py

**Comment by csadorf** [comment]
Thanks for reporting, looking into it now.

**Comment by csadorf** [comment]
@drvgautam The bug should now be fixed in version 0.3.1: https://pypi.org/project/marketplace-sdk/0.3.1/

**Comment by drvgautam** [comment]
@csadorf  thanks for fixing the issue.
